.eslintrc.js

```diff
@@ -310,6 +310,11 @@ module.exports = {
                         from: './src/**[!platform,telemetry,kernels,notebooks]**/**/*.ts',
                         message: 'Only modules from ./src/platform, ./src/telemetry and ./src/kernels can be imported into ./src/notebooks.'
                     },
+                    {
+                        target: './src/interactive-window/**/*.ts',
+                        from: './src/**webview**/**/*.ts',
```

Comment:
somehow `from: './src/**[!interactive-window,notebooks,kernels]**/**/*.ts'` doesn't, it always warns me of importing files inside `interactive-window`